Linux Script/configure-linux.sh

```diff
@@ -350,10 +353,15 @@ checkIfLogglyServersAccessible() {
   fi
 
   echo "INFO: Checking if '$LOGGLY_ACCOUNT' subdomain is valid."
-  if [ $(curl --head -s --request GET $LOGGLY_ACCOUNT_URL/login | grep "200 OK\|HTTP/2 200" | wc -l) ] >0; then
```

Comment:
@mchaudhary, At this point earlier, if I pass any invalid subdomain which doesn't contain **.loggly.com** with it but also does not exist, the script was marking it as a valid subdomain and showing it is reachable. This was happening because it checks at last if **wc -l > 0** then mark the subdomain valid and reachable. But actually what happens is if we pass any invalid subdomain which does not exist, the response is multiline which satisfies this condition and subdomain is marked as valid. 

**Note:** In both valid and invalid subdomain the response is in multiline so the script could not differentiate.